libgalois/src/GraphTopology.cpp

```diff
@@ -569,6 +582,27 @@ katana::ProjectedTopology::MakeTypeProjectedTopology(
     }
   });
 
+  katana::do_all(
+      katana::iterate(
+          (uint32_t)0, (uint32_t)std::ceil((double)topology.num_nodes() / 8.0)),
+      [&](uint32_t i) {
+        auto node_start = i * 8;
+        auto node_end = (i + 1) * 8;
+        node_end =
+            (node_end > topology.num_nodes()) ? topology.num_nodes() : node_end;
+        uint8_t val{0};
+        while (node_start != node_end) {
+          if (bitset_nodes.test(node_start)) {
```

Comment:
My point is that we can just store this bitset in the class if we really need to store it.

Reply:
I thought that the bitmask is required throughout as it stays with the node properties (same for edge properties). Maybe I am wrong

Reply:
Yes, you are right. I hadn't realized that.